### DISCUSSION TITLE
树形Table 或者树形数据 怎么支持

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1641] [Q&A 2.x]
像这种{a:"",children:[{a:""}]} 这种数据结构 怎么支持  能提供个思路吗

**Comment by liuweiGL** [comment]
没理解你想表达什么意思，你的原始需求是什么？

**Reply by Nokecy** [reply]
ArrayTable 可以添加子项，就是类似Table Tree ，像这种能不能提供一下思路
